src/test/resources/fixtures/test-fixtures/input-with-spellcheck-enabled/output.txt

```diff
@@ -0,0 +1,6 @@
+<div class="govuk-form-group">
+  <label class="govuk-label" for="input-with-spellcheck-enabled">
+    Spellcheck is enabled
+  </label>
+  <input class="govuk-input" id="input-with-spellcheck-enabled" name="spellcheck" type="text" spellcheck="true">
+</div>
```

Comment:
Is there also a test for input with no spellcheck defined? Is it worth adding one?

Reply:
Yeah, they won't be particularly clear in the diff.  One example is https://github.com/hmrc/play-frontend-govuk/blob/upgrade-to-3.8.0/src/test/resources/fixtures/test-fixtures/input-with-hint-text/input.json and there are other examples which start with `input-` in this directory https://github.com/hmrc/play-frontend-govuk/tree/upgrade-to-3.8.0/src/test/resources/fixtures/test-fixtures